integrations/dashly/README.md

```diff
@@ -0,0 +1,12 @@
+# analytics.js-integration-dashly [![Build Status][ci-badge]][ci-link]
+
+Dashly integration for [Analytics.js][].
+
+## License
+
+Released under the [MIT license](LICENSE).
```

Comment:
Why MIT? P. s. I'm not sure of the difference between licenses. I take MIT or BSD 3. But often can't tell why =)